packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -428,10 +451,32 @@ export function Item(props) {
 						onEdit={props.onEdit}
 						onChange={props.onChange}
 						className={props.nativeValueClassName}
+						wrap={nativeValueWrap}
 					/>
 				}
 				type={props.showType ? info.type : null}
 				tag={tag}
+				setWidth={lineItemWidth}
```

Comment:
Do you think it would be better to have a name like `isValueOverflown` ? I found it a little hard to know what this value was, and it's named like a setter.